### PR TITLE
variable sessionTime for sessionTokenData

### PR DESCRIPTION
**Pull request by lwin-kyaw**
- included `session_token_exp_second` in jrpc request to sss service to override default session token duration
- updated unittests
- update `http-helpers` module to v6

### CODE REVIEW

**Comment by lwin-kyaw** [comment]
The purpose of this PR is for mpcCoreKit to provide `custom sessionTime` based on clients' subscribed plans.
Ideal usage flow
- call TorusUtils static method, `setSessionTime` to provide custom sessionTime
- the above will override the default sessionTime (24hr) with the custom sessionTime provided
- include that sessionTime value in the request params of sss-service requests
- sss-service will accept the `sessionTime` to set the duration (expiry) of the `SESSION TOKEN` (https://github.com/torusresearch/sss-service/pull/88)

File: src/torus.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ import { Some } from "./some";
 // Implement threshold logic wrappers around public APIs
 // of Torus nodes to handle malicious node responses
 class Torus {
+  private static sessionTime: number = 2480; // 2480s = 24 hour
```
Comment (himanshuchawla009) [inline src/torus.ts:57]: this is not correct right, there are 86400s in 1 day.

**Review by himanshuchawla009 — CHANGES_REQUESTED** [review]

File: src/torus.ts (hunk 1 above)
Comment (lwin-kyaw) [reply src/torus.ts:57]: yes, yes. Thanks for pointing out.

**Review by himanshuchawla009 — APPROVED** [review]
lgtm

**Pull request merged**